File: view/app/src/main/java/com/example/tonjies/view/module/canvasx/ViewActivity.java
package com.example.tonjies.view.module.canvasx;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

import com.example.tonjies.view.R;
import com.example.tonjies.view.util.L;

/**
 *
 */
public class ViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_view);
        XView view = new XView(this);
    }
}

File: AView/appk/src/main/java/com/example/appk/module/canvasx/ViewActivity.kt
package com.example.appk.module.canvasx

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.example.appk.R

class ViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_view)
        val xView=XView(this)
    }
}